apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/__tests__/DailyScheduleTest.tsx

```diff
@@ -262,6 +262,25 @@ describe("fetchJourneys", () => {
       fetchJourneysMock.mockRestore();
     });
   });
+
+  it("fetches 62,76, and 627 when route is 627", () => {
```

Comment:
it's hard to tell in github, but looks like this should be moved up a few lines so it's within the describe("fetchJourneys")` block

Reply:
never mind I think my eyes are just playing tricks on me